src/Entity/User.php

```diff
@@ -1290,6 +1290,14 @@ public function setPersonalAccountDataRequestedAt(?\DateTimeImmutable $personalA
         $this->personalAccountDataRequestedAt = $personalAccountDataRequestedAt;
     }
 
+    public function canRequestPersonalAccountData(): bool
+    {
+        return match (true) {
+            $this->isBeneficiaire() => $this->telephone || $this->email,
+            default => false,
+        };
```

Comment:
`return $this->isBeneficiaire() ? $this->telephone || $this->email : false;`

Reply:
Plutot team Nils pour moi

Reply:
`return $this->isBeneficiaire() && $this->telephone || $this->email`